roles/openshift_gcp/tasks/provision_ssh_keys.yml

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
You'll need to include this from `roles/openshift_gcp/tasks/main.yml` with

```
- import_tasks: provision_ssh_keys.yml
```

so that provision, even when run individually, always has an SSH key.  `main.yml` is the only task file run automatically by ansible within a role.